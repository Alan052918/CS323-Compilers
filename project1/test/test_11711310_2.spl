int foo()
{
  a = 12;
  char b = '\x65';
}

char bar(int d)
{
  return $;
}

