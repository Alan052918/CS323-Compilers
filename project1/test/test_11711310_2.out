Error type B at Line 3: Missing specifier
Error type A at Line 9: unknown lexeme $
